docs/reference/commandline/system_df.md

```diff
@@ -0,0 +1,64 @@
+<!--[metadata]>
++++
+title = "system df"
+description = "The system df command description and usage"
+keywords = [system, data, usage, disk]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# system df
+
+```markdown
+Usage:	docker system df [OPTIONS]
+
+Show docker filesystem usage
+
+Options:
+      --help      Print usage
+  -v, --verbose   Show detailed information of space usage
+```
+
+The `docker system df` command displays information regarding the
+amount of disk space used by the docker daemon.
+
+By default the command will just show a summary of the data used:
+```bash
+$ docker system df
+TYPE                TOTAL               ACTIVE              SIZE                UNUSED
+Images              5                   2                   16.43 MB            11.63 MB (70%)
+Containers          2                   0                   212 B               212 B (100%)
+Local Volumes             2                   1                   36 B                0 B (0%)
+```
+
+A more detailed view can be requested using the `-v, --verbose` flag:
+```bash
+$ docker system df -v
+Images space usage:
+
+REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE                SHARED SIZE         UNIQUE SIZE         CONTAINERS
```

Comment:
We may need to explain `SIZE`, `SHARED SIZE` and `UNIQUE SIZE` somewhere (it's not clear to me what's what). If I `prune` will only the `UNIQUE` size be reclaimed?